toolkit/tools/internal/logger/writerhook.go

```diff
@@ -56,6 +58,13 @@ func (h *writerHook) Fire(entry *logrus.Entry) (err error) {
 		return
 	}
 
+	if h.useColors == false {
```

Comment:
Nit-pick to be consistent with what we do in the rest of our code. At least in most cases:
```suggestion
	if !h.useColors {
```